taxonomies/packaging_recycling.txt

```diff
@@ -32,14 +32,14 @@ pt:reutilizar, a reutilizar, para reutilizar, para ser reutilizado, a ser reutil
 en:recycle, to recycle, to be recycled, recyclable
 bg:за рециклиране, рециклирай, да се рециклира
 de:recyceln, zum Recyceln, recycelbar
-fr:recycler, à recycler, recyclable
+fr:bac de tri, recycler, à recycler, recyclable
 hu:újrahasznosítható, újrahasznosítandó, újrahasznosításra
 pt:reciclável, a reciclar, para reciclar
 # associated_materials: en:glass, en:aluminium, en:steel
 
 <en:recycle
 en:recycle in glass bin, recycle in glass container, recycle with glass, in glass bin, in glass container
-fr:recycler dans le conteneur à verre, recycler dans le bac à verre, recycler avec le verre, dans le conteneur à verre, avec le verre
+fr:tri verre, bac verre, recycler dans le conteneur à verre, recycler dans le bac à verre, recycler avec le verre, dans le conteneur à verre, avec le verre
```

Comment:
This one is tricky. We probably should keep uniform names, starting with the action "recycler dans le conteneur à verre", the others can be added as synonyms. Then we can find ways to make suggestions work for synonyms, and maybe have short name properties if we need to display them in a specific way.